openmdao/core/tests/test_check_totals.py

```diff
@@ -272,6 +281,111 @@ def compute_jacvec_product(self, inputs, d_inputs, d_outputs, mode):
                     d_inputs['in2'] += 5. * d_outputs['out2']
 
 
+class Simple(om.ExplicitComponent):
+    """
+    Simple matrix free component where some of/wrt partials are zero.
```

Comment:
is this docstring a copy/paste error? this comp doesn't appear to be matrix free...

Reply:
Yeah, bad docstring.  Fixed.